shpkpr/template.py

```diff
@@ -68,9 +68,10 @@ def render_json_template(template_file, **values):
     # build a new Jinja2 environment so we can inject some custom filters into
     # the template we're rendering.
     template_env = jinja2.Environment(undefined=jinja2.StrictUndefined)
+    template_env.filters['filter_items'] = template_filters.filter_items
     template_env.filters['require_int'] = template_filters.require_int
     template_env.filters['require_float'] = template_filters.require_float
 
     template = template_env.from_string(template_file.read())
-    rendered_template = template.render(**values)
+    rendered_template = template.render(_all_env=values, **values)
```

Comment:
Just remembered why this won't work: http://stackoverflow.com/a/3463669

TL;DR: The context is not available within a template as a named object, so in order to access it I'd have to add a function to the template that would return a context object when called, e.g. `context()` (like in the stack overflow example above).

Doing that exposes some extra issues that i'd rather avoid, like exposing all sorts of variables and functions that we don't necessarily want the developer to have access to at render time. Passing in a dict containing only the env vars seems like a nice compromise to me in terms of usability.